TODO1/levels-of-seniority.md

```diff
@@ -2,90 +2,90 @@
 > * 原文作者：[Kamran Ahmed](https://twitter.com/kamranahmedse)
 > * 译文出自：[掘金翻译计划](https://github.com/xitu/gold-miner)
 > * 本文永久链接：[https://github.com/xitu/gold-miner/blob/master/TODO1/levels-of-seniority.md](https://github.com/xitu/gold-miner/blob/master/TODO1/levels-of-seniority.md)
-> * 译者：
+> * 译者：[👊Badd](https://juejin.im/user/5b0f6d4b6fb9a009e405dda1)
 > * 校对者：
 
-# Levels of Seniority
+# 论资历的级别
 
-> How to Step Up as a Junior, Mid Level or a Senior Developer?
+> 如何成为初级、中级或高级开发人员？
 
-I have been working on redoing the [roadmaps](https://roadmap.sh) – splitting the skillset based on the seniority levels to make them easier to follow and not scare the new developers away. Since the roadmaps are going to be just about the technical knowledge, I thought it would be a good idea to reiterate and have an article on what I think of different seniority roles.
+一直以来，我都在重做[路径图](https://roadmap.sh) —— 根据资历级别划分技能，使它们更容易遵循，以免吓跑新入行的开发者。这些路径图只是关于技术知识的，所以我觉得有必要再重申一次，写一篇文章来表达我对不同资历的思考。
```

Comment:
"这些路径图只是关于技术知识的，所以我觉得有必要再重申一次，写一篇文章来表达我对不同资历的思考。" => "因为这些路径图只是关于技术知识的，所以我想有必要把它们回顾一遍并且写一篇文章来表达我对不同资历职位的思考。"

1. Since 因为……所以……; 2. Reiterate在这里用会更合适。强调了是回顾路径图，防止理解歧义。; 3. seniority roles对应资历职位。资历是在公司呆的时长，职位才是这里讨论的对象。